base/coreimg.jl

```diff
@@ -2,20 +2,24 @@
 
 Main.Core.eval(Main.Core, :(baremodule Inference
 using Core.TopModule, Core.Intrinsics
+import Core: print, println, show, write, unsafe_write, STDOUT, STDERR
+if false # show that the IO system is already (relatively) operational
+    print("HELLO")
+    println(" WORLD")
+    show("αβγ :)"); println()
+    println(STDERR, "TEST")
+    println(STDERR, STDERR)
+    println(STDERR, 'a')
+    println(STDERR, 'α')
+end
```

Comment:
It's disabled. I also added tests, although those don't run on quite as minimal an environment